gallery/src/styles/Nivo/NivoBarExample.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { ResponsiveBar, BarLegendProps } from '@nivo/bar';
+import { NIVO_COLORS } from '@sonatype/react-shared-components';
+import { rawBarData } from './data';
+
+const LEGENDS: BarLegendProps[] = [
+  {
+    dataFrom: 'indexes',
+    anchor: 'bottom-right',
+    direction: 'column',
+    itemWidth: 100,
+    itemHeight: 20,
+    translateX: 140
+  }
+];
+
+const barData = rawBarData.map(([id, value]) => ({ id, value }));
+
+const MyResponsiveBar = () => (
+  <div style={{ height: '400px' }}>
+    <ResponsiveBar data={barData}
+                   layout="vertical"
+                   colorBy="indexValue"
+                   enableLabel={false}
+                   margin={{
+                     top: 80,
+                     left: 60,
+                     bottom: 60,
+                     right: 140
+                   }}
+                   label={d => `${d.id}: ${d.value}`}
+                   padding={0.15}
+                   tooltipLabel={({ id }) => `${id}`}
```

Comment:
I'm not seeing any tooltips, is this doing anything?

Reply:
> I'm not seeing any tooltips, is this doing anything?

I thought it was showing the value when hovering over a bar, but that seems to work even without this: 788574dea623c5226bc44685d1514a67fd7582ce